CONTRIBUTING.md

```diff
@@ -352,7 +352,7 @@ for further information
 4.  The title (also called the subject) of your PR should be descriptive of your
     changes and succinctly indicates what is being fixed.
 
-    -   **Do not add the issue number in the PR title**.
+    -   **Do not add the issue number in the PR title or commit message.**.
```

Comment:
@FlameFractal please remove the extra period at the end of this line and then save the file, do a `git add [filename]`, then `git commit --amend`, then push the branch again with `-f`.

Thanks for catching the other spot to update this.